pylearner.ts

```diff
@@ -1633,14 +1740,16 @@ class MethodDeclaration extends AbstractMethodDeclaration {
     let outlineStartComment = null;
     let outlineStartEnv = null;
     let total = null;
-
+    let mayAliasesPreProofOutline: Immutable.Set<string> = ImmutableSet.of();
     for (let i = 0; i < this.bodyBlock.length; i++) {
       const stmt = this.bodyBlock[i];
       if (stmt instanceof ExpressionStatement && stmt.expr instanceof AssignmentExpression && stmt.expr.declaration != null) {
         const d = stmt.expr.declaration;
         env = EnvCons(d.getProofOutlineVariable(() => {
           return d.executionError(`Variabelen van type ${d.type.type!.toString()} worden nog niet ondersteund in bewijssilhouetten`);
         }), env);
+        if (outlineStart == null)
```

Comment:
Wat als je meerdere proof outlines na elkaar hebt? Dan neem je de aliasing die tot stand komt tijdens de eerste proof outline niet in rekening voor de tweede proof outline.